shimmy/__init__.py

```diff
@@ -45,22 +52,15 @@ def __call__(self, *args: list[Any], **kwargs: Any):
         e,
     )
 
-try:
```

Comment:
Why did you change this?

Reply:
Oh that must’ve gotten removed accidentally due to all the weird branching and such. Will fix it now.